### PR TITLE
update

### PR DESCRIPTION
**Pull request by nawnie**
add sample_prompts.json

streamline load from zip (other compressions are a wip (huge time saving when loading from a zip in gdrvie))

add more sampler schedules

### CODE REVIEW

**Review by nawnie — COMMENTED** [review]
reset defaults

File: Train_Colab.ipynb [hunk 1]
```diff
@@ -80,12 +104,24 @@
...
+          "base_uri": "https://localhost:8080/"
+        }
       },
-      "outputs": [],
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "DONE!\n"
```
Comment (victorchall) [inline Train_Colab.ipynb:121]: Need to clear the output

File: Train_Colab.ipynb [hunk 2]
```diff
@@ -47,25 +47,49 @@
...
+          "base_uri": "https://localhost:8080/"
+        }
       },
-      "outputs": [],
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Mounted at /content/drive\n"
```
Comment (victorchall) [inline Train_Colab.ipynb:89]: Need to clear output.

File: Train_Colab.ipynb [hunk 3]
```diff
@@ -47,25 +47,49 @@
...
+          "base_uri": "https://localhost:8080/"
+        }
       },
-      "outputs": [],
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Python 3.10.6\n"
```
Comment (victorchall) [inline Train_Colab.ipynb:64]: Need to clear output

File: Train_Colab.ipynb [hunk 4]
```diff
@@ -343,27 +380,31 @@
...
-        "#@markdown * Ratio of training on empty caption. Improves unconditional guidance.\n",
-        "Conditional_DropOut = 0.04 #@param {type:\"slider\", min:0, max:0.3, step:0.01}\n",
-        "#@markdown * Ratio of images randomly to flip horizontally.  Use for small data sets. May negtively affect likeness of subjects with asymmetric features.\n",
-        "Picture_flip = 0.0 #@param {type:\"slider\", min:0, max:0.5, step:0.05}\n",
-        "#@markdown * This can improve contrast in light and dark scenes, Use a ratio between 0-10% for Best results.  0\n",
-        "zero_frequency_noise = 0.05 #@param {type:\"slider\", min:0, max:0.25, step:0.01}\n",
+        "#@markdown * Skip the nth last layer of CLIP.\n",
+        "Clip_skip = 1 #@param {type:\"slider\", min:0, max:4, step:1}\n",
+        "#@markdown * ratio of captions dropped from training data.\n",
+        "Conditional_DropOut = 0.1 #@param {type:\"slider\", min:0, max:0.3, step:0.01}\n",
```
Comment (victorchall) [inline Train_Colab.ipynb:386]: Probably should default dropout back to a lower value like 0.04, 0.10 is a bit high from my experience for the typical small training?

File: Train_Colab.ipynb [hunk 5]
```diff
@@ -343,27 +380,31 @@
...
+        "#@markdown * Ratio of images randomly to flip horizontally.\n",
+        "Picture_flip = 0.05 #@param {type:\"slider\", min:0, max:0.5, step:0.05}\n",
```
Comment (victorchall) [inline Train_Colab.ipynb:388]: I think flip should either be zero or 0.50, I recommend leaving it 0.0 as thats the default for the CLI trainer.

File: utils/sample_generator.py [hunk 6]
```diff
@@ -272,13 +272,25 @@ def create_inference_pipe(self, unet, text_encoder, tokenizer, vae, diffusers_sc
     @torch.no_grad()
     def _create_scheduler(self, scheduler_config: dict):
         scheduler = self.scheduler
-        if scheduler not in ['ddim', 'dpm++']:
+        if scheduler not in ['ddim', 'dpm++', 'pndm', 'ddpm', 'lms', 'euler', 'euler_a', 'kdpm2']:
```
Comment (victorchall) [inline utils/sample_generator.py:275]: Awesome!

**Comment by victorchall** [comment]
Nice work!

I do notice some defaults keep getting changed, recommend when you develop to watch for that.  Changing defaults on updates will lead to inconsistent results for users.
